oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -587,7 +587,8 @@ export default class Map extends React.PureComponent {
         if (feature.get("type") != null) {
           switch(feature.get("type")) {
             case "class4":
-              content.push(feature.get("id"));
+              const class4id = feature.get("id").replace("/", "_");
```

Comment:
I think a brief comment in the code explaining why we're doing the replacement would be good for the future.